examples/MMF-activity-recognition/README.md

```diff
@@ -0,0 +1,150 @@
+
+For the activity recognition MMF model, we need to provide a custom handler. The handler generally extends the [Base handler](https://github.com/pytorch/serve/blob/master/ts/torch_handler/base_handler.py). The [handler](https://github.com/pytorch/serve/blob/adding_MMF_example/examples/MMF-activity-recognition/handler.py) for MMF model, needs to load and intialize the model in the initialize method and then in the preprocess, mimics the logic in dataset processors to make a sample form the input video and its realted text ( preprocess the video and make the related tensors to video,audio and text). The inference method run the preprocessed samples through the  MMF model and send the outputs to the post process method. 
+
+To intialize the MMF model in the [intialization method](https://github.com/pytorch/serve/blob/adding_MMF_example/examples/MMF-activity-recognition/handler.py#L65), there are few poinst to consider. We need to load config (Line#91) using OmegaConfing, then setup_very_basic_config() (Line#93) function from mmf.utils.logger    and setup_imports()  (Line#95) from  mmf.utils.env need to be called to setup the enviroment for loading the model. Finally to load the model (Line#96), we pass the model config to the  [MMFTransformer](https://github.com/facebookresearch/mmf/blob/video_datasets/mmf/models/mmf_transformer.py) model. 
+
+##### Getting started on Serving
+
+To make the .mar file in the current setting we will use model_archiver as follow. ( if you have already have downloaded the .mar file you can skip this step)
+
+Make sure to compress configs, datasets and models folder.
+
+```
+torch-model-archiver --model-name MMF_model --version 1.0 --serialized-file $(path to checkpoints) --handler handler.py --extra-files "charades.csv,mmf_utils.py,transforms.py,$(path to the aggreagted config file,saved in mmf/save/config.yaml)"
+```
+
+charades.csv has been created using the [notebook](https://github.com/pytorch/serve/blob/adding_MMF_example/examples/MMF-activity-recognition/Generting_Charades_action_lables.ipynb) and is available here as well, which contains the labels from the dataset to be used for setting the model ouput shape and also index to class mapping. Running this will result in MMF_model.mar in the current directory.
+
+The next step is to make a model_store and move the .mar file to it:
+
+```
+mkdir model_store
+mv MMF_model.mar model_store
+```
+
+Now we can start serving our model:
+
+```
+torchserve --start --model-store model_store
+curl -X POST "localhost:8081/models?model_name=MMF_model&url=MMF_model.mar&batch_size=1&max_batch_delay=5000&initial_workers=1&synchronous=true"
+```
+
+Sending inference request using
+
+The examples of video and info.json is avalilbe here for demonstration purposes. **Please make sure to set the corret path to the video and info.json in the request.py.**
+
+```
+Python request.py
+```
+
+This will write the results in response.txt in the current directory.
+
```

Comment:
You don't need full charades.csv file, you only need the label mapping. Maybe simplify this to only use label mapping?

Reply:
@apsdehal changed the csv file to only include the labels.